DOCKER.md

```diff
@@ -218,6 +219,15 @@ chmod 400 nginx/*.key
 
 ```docker build  -t defectdojo/defectdojo-nginx -f Dockerfile.nginx .```
 
+* Run defectDojo with: 
+```
+rm -f docker-compose.override.yml
+ln -s docker-compose.override.https.yml docker-compose.override.yml
+docker-compose up
```

Comment:
this appears twice. can you put it once only please? maybe create two sections with the pre-requisites for: 
- using a self-signed certificate
- using your own certificate (where to copy the certificate, set generate_certificate=False)

Then describe the common part to both (start with the https override)